verilog/tools/kythe/indexing_facts_tree_extractor.cc

```diff
@@ -1284,7 +1291,9 @@ void IndexingFactsTreeExtractor::ExtractInclude(
                          IndexingFactType::kInclude));
   } else {
     std::string content;
-    if (!TryToReadFile(file_path, content, filename, include_dir_paths_)) {
+    if (!SearchForFileAndGetContents(file_path, content, filename,
+                                     include_dir_paths_)
+             .ok()) {
```

Comment:
This function is called in the switch statement and return `status` here won't benefit at the moment because we need to propagate the statuses up to the binary.
I suggest leaving this until be add a proper `status` which reports about the status of the binary overall.